src/syscall_sdl.c

```diff
@@ -7,13 +7,311 @@
 #error "Do not manage to build this file unless you enable SDL support."
 #endif
 
+#include <pthread.h>
 #include <stdint.h>
 #include <stdio.h>
+#include <unistd.h>
 
 #include <SDL.h>
+#include <SDL_mixer.h>
 
 #include "state.h"
 
+/* Copyright (C) 2021-2022 Steve Clark
```

Comment:
s/Copyright/Portions Copyright/

Reply:
Rearrange these newly-added lines immediately preceding the function `sfx_handler` for enhanced readability.